Dockerfile

```diff
@@ -67,7 +67,7 @@ RUN apt-get update -y -q && \
     apt-get install -y clang-3.9 lldb-3.9 && \
     apt-get install -y libc6-i386  libsuitesparse-dev && \
     export LLVM_CONFIG=/usr/lib/llvm-3.9/bin/llvm-config && \ 
-    git clone --recursive https://github.com/simonsfoundation/CaImAn.git && \
+    git clone --recursive https://github.com/ecobost/CaImAn.git && \
```

Comment:
Don't you want to make a PR to the original one and we'll use that? The problem with using your version is, that it's harder to get updates.